src/context/polling.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useContext, useEffect, useState } from "react"
```

Comment:
suggestion regarding file name: since it is only batch job related polling file and the provider could be renamed (and/or) moved closer to the domain logic

Reply:
I wasn't sure if it would end up containing only stuff related to batch jobs. wdyt? should I move it for now and see later what happens? 

Just for your information, I will do a write up about the notification and it could be possible that we poll the notifications and not the batch jobs if it is accepted, which means that it would not be linked to the batch jobs anymore but to the notifications and in that case this provider make more sense

Reply:
yes, it makes sense to leave it here if it will be a generic polling context, but if it stays related to the batch job domain, it's better to move it somewhere "closer" to the usecase